presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -233,6 +233,7 @@ public JdbcTableHandle getTableHandle(JdbcIdentity identity, SchemaTableName sch
                 while (resultSet.next()) {
                     JdbcTypeHandle typeHandle = new JdbcTypeHandle(
                             resultSet.getInt("DATA_TYPE"),
+                            resultSet.getString("TYPE_NAME").contains("UNSIGNED"),
```

Comment:
Will `UNSIGNED` be always in capital letters.
Reading through some of the source code for `DatabaseMetaData` implementation in mysql connector code it seems [getCapitalizeTypeNames](https://github.com/spullara/mysql-connector-java/blob/master/src/main/java/com/mysql/jdbc/ConnectionProperties.java#L128) is used to check whether to convert to upper case. 
Is a conversion to uppercase warranted before the contains check? . Can you try testing with this flag turned off.

Reply:
how about use `toUpperCase()`?
`getString("TYPTE_NAME").toUpperCase().contains("UNSIGNED")`

Reply:
yeah. converting to upper case sounds better. Let's do that.